src/media/js/settings_app.js

```diff
@@ -19,6 +19,9 @@ define('settings_app',
         return window.location.search.indexOf('cache=false') === -1;
     }
 
+    var is_packaged_app = window.location.protocol === 'app:';
```

Comment:
I've opened mozilla/marketplace-core-modules/pull/33 for this but we can start in settings and clean up afterwards.